### ISSUE TITLE
UI Transform- New tables added are hidden behind the firsr added table. 

### ISSUE THREAD

**Issue by nairabhishek73** [DalgoT4D/webapp#589]
https://jam.dev/c/e322e796-f288-4667-b582-02d050e5cc59

New tables should be added via a '+' button 
New tables should appear below the original table and be visible on addition

**Comment by nairabhishek73** [comment]
1) Plus button is added- but all the plusses are misaligned. Limit text to a point so that all plusses can be aligned. 
- If the store folder can be eliminated and just have a folder of every schema then that might help create amore space (Let me know if i should open a separate issue for this)
2) The newly added table is still hidden behind the currrent table.

https://jam.dev/c/a968065d-8127-42dd-a6ee-e6cf43a54b96

**Comment by fatchat** [comment]
@mdshamoon

**Comment by nairabhishek73** [comment]
Hi please check: https://jam.dev/c/514e0f99-c0b4-473c-a9cd-3a2716cec6cc

**Comment by fatchat** [comment]
the issues get completed as part of PR merges (remember this from september?)
at some point we will figure out how to close only after the code gets released
please check now

**Comment by nairabhishek73** [comment]
After update, table appearing to the right, should appear below as per figma since flow is generated left to right.

**Comment by fatchat** [comment]
hmm @mdshamoon is this easy to do?

@nairabhishek73 i'm removing the "high priority" tag

**Comment by mdshamoon** [comment]
Yes @fatchat. Will update it to move below the last added node

**Comment by fatchat** [comment]
@nairabhishek73 ready to close?

**Comment by nairabhishek73** [comment]
Yes!
